common/config/babel/.babelrc.js

```diff
@@ -48,7 +48,9 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // feature for blurred background and replace background effects
         'video-background-effects',
         // feature for vertical gallery layouts in VideoGallery
-        'vertical-gallery'
+        'vertical-gallery',
+        // Data loss prevention feature
+        'dlp'
```

Comment:
nitpick: Looks like the start of the list was kept in alphabetical order...and descends into not. Might be worth at least putting your `dlp` in alphabetical order

Reply:
I think it used to be in alphabetical order, but it ended not being the case so just adding at the end, will change the order when i make another commit